prow/scripts/job-waiter.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+prowJobId=${PROW_JOB_ID}
+requiredJobLabelKey=${REQUIRED_JOB_LABEL_KEY:-kyma-job-type}
+requiredJobLabelValue=${REQUIRED_JOB_LABEL_VALUE:-component}
+maxRetries=${MAX_RETRIES:-100}
+
+eventGUID=$(kubectl get pod "${prowJobId}" -ojson | jq -r '.metadata."labels"."event-GUID"')
+
+echo "Prow Job ID: ${prowJobId}"
+echo "GUID: ${eventGUID}"
+echo "Required label: ${requiredJobLabelKey}=${requiredJobLabelValue}"
+echo "Max retries: ${maxRetries}"
```

Comment:
as an input, we should accept the deadline in minutes. Now you configure maxRetries to 100, but then you need to know that you sleep 5 seconds between every query.

Reply:
Sure, it was just the simplest version possible. The Go version of this utility will have timeout in minutes.